src/main/java/app/linguistai/bmvp/service/wordbank/UnknownWordService.java

```diff
@@ -343,6 +345,60 @@ public ROwnerUnknownWordList deleteList(UUID listId, String email) throws Except
         }
     }
 
+    @Transactional
+    public ROwnerUnknownWordList addPredefinedWordList(String wordListYamlFile, String email) throws Exception {
```

Comment:
Removing the `wordListYamlFile` parameter and using the defined constant insted in the function will be better in terms of the principle of least privilege, potentially avoiding some problems

Reply:
Keeping it so we can add more predefined word lists in the future, for IELTS etc.